src/components/app-header.js

```diff
@@ -211,6 +211,45 @@ class AppHeader extends React.PureComponent {
         }
       }
 
+      else if (mode === 'public') {
+
+        // If mode is public then render insti logo 
+        if (institute.imagery.logo) {
```

Comment:
Priority should be to show site logo in the left if available. 

If the reason to do this was the SHP publish, that case should be handled on the publish script.

Reply:
All other cases would be affected once priority is given to the site logo